### DISCUSSION TITLE
Error 404 when requesting for latest obeservations for a valid station

### DISCUSSION THREAD

**Discussion by rosteve** [weather-gov/api#252] [Q&A]
**Describe the bug**
Is it expected to get error 404 when requesting for latest obeservations for a valid station?
Ideally, the result should be an empty list, not error 404 as its currently returning.

**To Reproduce**
To reproduce, use this API endpoint for SIOUX CENTER (COOPSIXI4): 

https://api.weather.gov/stations/COOPSIXI4/observations/latest

Here's the response for the above request.
{
    "correlationId": "5316761a",
    "title": "Not Found",
    "type": "https://api.weather.gov/problems/NotFound",
    "status": 404,
    "detail": "Not Found",
    "instance": "https://api.weather.gov/requests/5316761a"
}

**Expected behavior**
It should return an empty list and not error 404.

**Environment**
Google Chrome Version 87.0.4280.88 (Official Build) (x86_64)

**Comment by jah1007** [comment]
While it is a valid station, no data is currently being sent from the co-op site.

**Comment by scadergit** [comment]
The /latest endpoint does not return a list, but a record. As @jah1007 noted, the record doesn't exist and returns 404. Whereas https://api.weather.gov/stations/COOPSIXI4/observations is a list and does return empty.
